open_academy/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import controllers
```

Comment:
Controllers are not needed at this point, so remove the import and also the folder